src/modules/search-result/search-result.jsx

```diff
@@ -29,6 +29,7 @@ const SearchResult = ({
   children,
   id,
   className,
+  tag: Tag = Card,
```

Comment:
I don't think this should work so. You can support tag but it's better to pass it forward to the Card.

What is your use case for this?

Reply:
I have a component [Search.Result](https://github.com/oacore/search/blob/master/modules/search-layout/result.jsx) and I need to put SearchResult from here to it because it applies all grid styles I need otherwise I would need to import css styles modules and that's not good too.

Reply:
I don't get it. If you put this SearchResult as a direct child and add one more class from your component, it would work, wouldn't it?

Reply:
yes it would but I don't see anything wrong doing it with Tag. It's one node less.

Reply:
Card component in the root of the module is a part of the design that is why I don't like the idea. The component should not become something else than a Card when the element type itself could be changed.

Reply:
so basically if I keep tag in `....htmlProps` and pass it down to Card it's ok with you?

Reply:
Yes, I considered such an option when wrote the component. You can change the actually rendered component/element but card classes will be always passed.